de.fu_berlin.inf.dpp/src/de/fu_berlin/inf/dpp/StartupSaros.java

```diff
@@ -141,7 +140,8 @@ private void startSTFController(final int port) {
             @Override
             public void run() {
                 try {
-                    STFController.start(port, context);
+                    // TODO FIX Project dependency
```

Comment:
This is the reason why the STF tests are not executable anymore.
If you do something like that, you have to highlight this in your commit message. Furthermore you have to trigger a discussion about this topic instead of merging the PR as soon as possible !